Library/Homebrew/extend/ARGV.rb

```diff
@@ -88,11 +88,9 @@ def kegs
             Formulary.from_rack(rack)
           end
 
-          if (prefix = f.installed_prefix).directory?
-            Keg.new(prefix)
-          else
-            raise MultipleVersionsInstalledError, rack.basename
-          end
+          raise MultipleVersionsInstalledError, rack.basename unless (prefix = f.installed_prefix).directory?
```

Comment:
Better to try and keep this below 80 chars by e.g. splitting this into an `unless .. end`.